tensorflow/core/kernels/uniform_quant_ops/uniform_quantize_op_test.cc

```diff
@@ -22,6 +22,53 @@ namespace tensorflow {
 
 class UniformQuantizeOpsTest : public OpsTestBase {
  protected:
+  struct QParams {
+    QParams(int q_axis, int q_min_val, int q_max_val, std::vector<float> scale,
+            std::vector<int32> zp)
+        : q_axis(q_axis),
+          q_min_val(q_min_val),
+          q_max_val(q_max_val),
+          scale(scale),
+          zp(zp) {}
+    int q_axis;
+    int q_min_val;
+    int q_max_val;
+    std::vector<float> scale;
+    std::vector<int32> zp;
+  };
+
+  template <typename Tin, typename Tout>
+  void TestUniformQuantize(std::vector<Tin> &input, std::vector<long> &shape,
+                           QParams &q_params,
+                           std::vector<Tout> &expected_result) {
+    TF_ASSERT_OK(NodeDefBuilder("test", "UniformQuantize")
+                     .Input(FakeInput(DataTypeToEnum<Tin>::v()))
+                     .Input(FakeInput(DT_FLOAT))
+                     .Input(FakeInput(DT_INT32))
+                     .Attr("Tin", DataTypeToEnum<Tin>::v())
+                     .Attr("Tout", DataTypeToEnum<Tout>::v())
+                     .Attr("quantization_axis", q_params.q_axis)
+                     .Attr("quantization_min_val", q_params.q_min_val)
+                     .Attr("quantization_max_val", q_params.q_max_val)
+                     .Finalize(node_def()));
+    TF_ASSERT_OK(InitOp());
+
+    const long scale_size = static_cast<long>(q_params.scale.size());
+    TensorShape scale_shape(scale_size > 1 ? TensorShape({scale_size})
+                                           : TensorShape({}));
+    const long zp_size = static_cast<long>(q_params.zp.size());
+    TensorShape zp_shape(zp_size > 1 ? TensorShape({zp_size})
+                                     : TensorShape({}));
+    AddInputFromArray<Tin>(TensorShape(shape), input);
```

Comment:
This line is causing a compilation failure on Windows. TensorShape's constructor takes a span of int64_t values, but on Windows, `long` is 32-bit. The Google style-guide [prohibits](https://google.github.io/styleguide/cppguide.html#Integer_Types) the use of types like `long` for this reason.

Can you replace uses of `long` with `int64_t`?

Reply:
Thanks for pointing that out. I have replaced it.